docs/pages/x.tsx

```diff
@@ -3,22 +3,24 @@ import Divider from '@mui/material/Divider';
 import Head from 'docs/src/modules/components/Head';
 import BrandingCssVarsProvider from 'docs/src/BrandingCssVarsProvider';
 import AppHeader from 'docs/src/layouts/AppHeader';
+import AppFooter from 'docs/src/layouts/AppFooter';
+import AppHeaderBanner from 'docs/src/components/banner/AppHeaderBanner';
+import References, { ADVANCED_CUSTOMERS } from 'docs/src/components/home/References';
 import XHero from 'docs/src/components/productX/XHero';
 import XComponents from 'docs/src/components/productX/XComponents';
 import XDataGrid from 'docs/src/components/productX/XDataGrid';
+// import XPlans from 'docs/src/components/productX/XPlans';
```

Comment:
```suggestion
// import XPlans from 'docs/src/components/productX/XPlans';
```

Why creating `XPlans2` instead of rewriting `XPlans`?

Reply:
Created a new file just to experiment before overriding the existing one! 
Let me remove these dead pieces, though 😃